digital_assets/aswf_digital_assets_license.txt

```diff
@@ -0,0 +1,17 @@
+ASWF Digital Assets License v1.0
+
+License for <Asset Name, e.g. “Moana Island Scene”> (the “Asset Name”).
```

Comment:
I'd remove "Moana Island Scene" -- so changing this line to just say:

`License for <Asset Name> (the “Asset Name”).`

Reply:
Would also make the same change in the version that gets posted at aswf.io

Reply:
Done - note I'm thinking the URI for aswf.io will just be a forward to this version. You think that will be ok?